persistence/postgres-persistence/src/main/java/com/netflix/conductor/postgres/config/PostgresProperties.java

```diff
@@ -13,13 +13,16 @@
 
 import java.time.Duration;
 import java.time.temporal.ChronoUnit;
-
+import org.springframework.beans.factory.annotation.Value;
 import org.springframework.boot.context.properties.ConfigurationProperties;
 import org.springframework.boot.convert.DurationUnit;
 
 @ConfigurationProperties("conductor.postgres")
 public class PostgresProperties {
 
+    @Value("${conductor.outbox.table.enabled}")
+    protected boolean outboxEnabled;
```

Comment:
make this private with a getter, just like the other properties to preserve encapsulation

Reply:
done